examples/wallets/examples/keystore_signer.rs

```diff
@@ -0,0 +1,48 @@
+//! Example of using a keystore wallet to sign and broadcast a transaction on a local Anvil node.
+
+use alloy::{
+    network::EthereumSigner,
+    node_bindings::Anvil,
+    primitives::{address, U256},
+    providers::{Provider, ProviderBuilder},
+    rpc::{client::RpcClient, types::eth::request::TransactionRequest},
+    signers::wallet::Wallet,
+};
+use eyre::Result;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Spin up an Anvil node.
+    let anvil = Anvil::new().block_time(1).try_spawn()?;
+
+    // Password to decrypt the keystore file with.
+    let password = "test";
+
+    // Set up a wallet using Alice's keystore file.
+    // The private key belongs to Alice, the first default Anvil account.
+    let wallet =
+        Wallet::decrypt_keystore("./examples/wallets/examples/keystore/alice.json", password)?;
```

Comment:
this needs to use manifest dir + join from env var so this can be run from anywhere

Reply:
Added `PathBuf::from(env::var("CARGO_MANIFEST_DIR")?).join("examples/keystore/alice.json")`

This should join from the `Cargo.toml` location in the `examples/wallets/` crate